src/cplscheme/SerialCouplingScheme.cpp

```diff
@@ -87,38 +87,42 @@ void SerialCouplingScheme::performReceiveOfFirstAdvance()
   }
 }
 
-bool SerialCouplingScheme::exchangeDataAndAccelerate()
+void SerialCouplingScheme::exchangeFirstData()
 {
-  bool convergence = true;
-
   if (doesFirstStep()) { // first participant
     PRECICE_DEBUG("Sending data...");
     sendTimeWindowSize();
     sendData(getM2N(), getSendData());
-    PRECICE_DEBUG("Receiving data...");
+    PRECICE_DEBUG("Receiving convergence data...");
     if (isImplicitCouplingScheme()) {
-      convergence = receiveConvergence(getM2N());
+      receiveConvergence(getM2N());
```

Comment:
Seems wrong that we `receiveConvergence` here and not in `exchangeSecondData`.

Reply:
Good catch!